apps/core/files/base.py

```diff
@@ -220,3 +220,6 @@ def fast_transfer_file(
 
 class BaseStorage(StorageFileOverwriteMixin, BkJobMixin, Storage, ABC):
     storage_type: str = None
+
+    def get_file_md5(self, file_name: str) -> str:
+        raise NotImplementedError
```

Comment:
`raise NotImplementedError()`

Reply:
https://stackoverflow.com/questions/16706956/is-there-a-difference-between-raise-exception-and-raise-exception-without